README.md

```diff
@@ -154,6 +154,35 @@ kubectl delete namespace kogito-workflows
 make undeploy
 ```
 
+## Use Workflow Development Mode
+In the development mode, a user can edit and reload the files using the quarkus dev mode.
+To enable the dev mode add this annotation in the KogitoServerlessWorkflow CR:
+```
+sw.kogito.kie.org/profile: dev
+```
+
+In development mode, different external files can be edited. Each file type will be saved into a different configmap.  
+Currently we support those types:
+- CamelRoute
+- OpenAPI
+- Async API
+- Generic: any other type
+
+To be able to edit a file via a configmap, you must setup an annotation which will reference the specific configmap previously created on the cluster.
+
+```
+  sw.kogito.kie.org/resource-camel: mycamel-configmap
+  sw.kogito.kie.org/resource-openapi: myopenapi-configmap
+  sw.kogito.kie.org/resource-asyncapi: myasyncapi-configmap
+  sw.kogito.kie.org/resource-generic: mygeneric-configmap
+```
+Each annotation entry will be turned into a file inside the path `src/main/resources/<type specific folder>` and using the key as the filename.
```

Comment:
doc matter: Where can I find the `type specific folder` ?

Reply:
Added